module/VuFind/src/VuFind/ILS/Driver/Folio.php

```diff
@@ -1284,7 +1290,17 @@ public function getInstructors()
      */
     public function getCourses()
     {
-        return $this->getCourseResourceList('courses');
+        $showCodes = $this->config['CourseReserves']['displayCourseCodes'] ?? false;
+        $courses = $this->getCourseResourceList(
+            'courses',
+            null,
+            $showCodes ? ['courseNumber', 'name'] : ['name'],
+            $showCodes ? '%s: %s' : '%s'
+        );
+        $callback = function ($course) {
+            return trim(ltrim($course, ':'));
```

Comment:
This is nice. I feel like when we forget to do this, there can be problems.

Reply:
The `ltrim` also takes care of the case where the course number field is blank. :-)